pages/githubissues.md

```diff
@@ -41,7 +41,7 @@ If you're still confused, that's quite alright. The data flow of GitHub can be q
 ![GitHub repos flowchart](uploads/images/flowchart_rep_fork_wiki.png)
 
 ## Create a Commit
-After you are done making your changes use the command `git status`. If the list of files changed only includes those you wish to modify, use the command `git add .` (The '.' is part of the command.) Otherwise you can also choose only certain changes to include by using `git add <file1> <file2> <file3>...`  Now that you have selected the files you wish to include, use the commands `git commit -m "commit message"` and `git push -u origin name_of_your_new_branch` to save your changes and push them to your Github (for any further commit on the same branch after the first one, you can just use `git push`).
+After you are done making your changes use the `git status` command to see which files in the working directory have been modified. If you wish to stage all of the modified files shown use the `git add .` (The '.' is part of the command.) command. Otherwise you can choose only the file or files you wish to stage by using the `git add file1, file2, file3...` command. Now that you have selected the files you wish to include, use the commands `git commit -m "commit message"` and `git push -u origin name_of_your_new_branch` to save your changes and push them to your Github (for any further commit on the same branch after the first one, you can just use `git push`).
```

Comment:
- Would adding comma after "After you are done making your changes" and "If you wish to stage all of the modified files" make those two sentences better? 
- Since `git add file1, file2, file3...` is formatted as inline code in markdown, would it be better to not add comma between `files1`, `file2`, and `file3`?

Reply:
@xyb994 I'll make the changes
.